### PR TITLE
add tenderly tutorial

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Please explain the changes this PR addresses here.

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: tutorials/eth-api/using-tenderly.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+There is a limit of one project with a free account, however, you can have multiple smart contracts under the purview of a single project.
+
+## Add Smart Contracts {: #add-smart-contracts }
+
+Adding a smart contract to your Tenderly project is akin to bookmarking it. While not required, adding a contract will unlock additional Tenderly features over simply searching for the contract in the Tenderly platform. 
+
+To add a smart contract to your Tenderly project, Click on the **Contracts** tab under the Inspect heading, then Click **Add Contracts**. Then, take the following steps:
+
+1. Enter the address of the contract. For this tutorial, we'll be using the Multichain USDC contract `0x818ec0a7fe18ff94269904fced6ae3dae6d6dc0b` 
+2. Select the network the contract is deployed to. We'll select Moonbeam in this case
```
Comment (eshaben) [inline tutorials/eth-api/using-tenderly.md:50]:
```suggestion
2. Select the network the contract is deployed to. We'll select **Moonbeam** in this case
```

File: tutorials/eth-api/using-tenderly.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+3. Give the contract a name to help you recognize it in the dashboard
+4. Press **Add Contract** 
+
+![Add a smart contract](/images/tutorials/eth-api/using-tenderly/tenderly-2.png)
+
+## Simulate a Transaction {: #simulate-a-transaction }
+
+Simulations allow you to see how a transaction will execute without actually sending it to execute on the blockchain. You can simulate a transaction against any point in time, or simply the latest block. 
+
+Let's craft a transaction to simulate against the Moonbeam network by taking the following steps:
```
Comment (eshaben) [inline tutorials/eth-api/using-tenderly.md:60]: this jumps over to simulations while the user is still on the contracts tab, so just add in that they have to switch to the Simulator tab

File: tutorials/eth-api/using-tenderly.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+3. Next, we'll input relevant function parameters. For destination address you can input any address, such as Alith's address: `0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac` 
+4. For amount, you can also specify any amount, such as `10000000000`
+5. Select **Pending Block** to run the simulation against the latest Moonbeam block produced 
+6. Specify the from address as Baltathar: `0x3Cd0A705a2DC65e5b1E1205896BaA2be8A07c6e0` or another address of your choice
+7. Press **Simulate Transaction**
+
+
+![Simulate a transaction against Moonbeam](/images/tutorials/eth-api/using-tenderly/tenderly-3.png)
+
+Clearly, this simulated transaction is going to fail because we're trying to send 10,000 USDC that we don't have. But, with [Tenderly simulator](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction){target=_blank}, we can tinker with the blockchain state and run simulations that assume different conditions. For example, let's run the simulation assuming that Baltathar actually holds a balance of 10,000 USDC. To do so, take the following steps:
```
Comment (eshaben) [inline tutorials/eth-api/using-tenderly.md:73]: might want to add that the user will need to click Re-Simulate in the top right corner

File: tutorials/eth-api/using-tenderly.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+## Fork the Chain {: #fork-the-chain }
+
+### Create a Fork {: #create-a-fork }
+
+Simulations are great for one-off tests, but what if you want to test a series of interdependent transactions? A fork is a better option in this case because forks are stateful. Additionally, [Tenderly forks](https://docs.tenderly.co/simulations-and-forks/forks){target=_blank} are a terrific option when you want to interact with contracts in a private environment, without re-deploying existing smart contract logic on-chain. 
+
+!!! note
+    There are some limitations to be aware of when using Tenderly's forking feature. You cannot interact with any of the [Moonbeam precompiled contracts](/builders/pallets-precompiles/precompiles/){target=_blank} and their functions. Precompiles are a part of the Substrate implementation and therefore cannot be replicated in the simulated EVM environment. This prohibits you from interacting with cross-chain assets on Moonbeam and Substrate-based functionality such as staking and governance.
+
+It's easy to create a fork with Tenderly. To do so, take the following steps:
```
Comment (eshaben) [inline tutorials/eth-api/using-tenderly.md:102]: users might still be on the simulator tab, so just a reminder to switch to forks would be nice

File: tutorials/eth-api/using-tenderly.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+1. Select the desired contract to interact with.
+2. Select the contract function you'd like to call. We'll choose `mint` in this case.
+3. Enter Baltathar's address: `0x3Cd0A705a2DC65e5b1E1205896BaA2be8A07c6e0`
+4. Enter the amount to mint, such as `10000000000000`
+5. Minting is a privileged role in the USDC contract. We need to specify the sender of this transaction in our fork as the [authorized minter of the USDC contract](https://moonbeam.moonscan.io/token/0x818ec0a7fe18ff94269904fced6ae3dae6d6dc0b#readContract){target=_blank}, which is `0xEC4486a90371c9b66f499Ff3936F29f0D5AF8b7E`
+6. Press **Simulate Transaction**
+
+![Run simulation on fork to mint USDC](/images/tutorials/eth-api/using-tenderly/tenderly-6.png)
+
+Great! Now let's go ahead and try to perform the transfer from Baltathar now that Baltathar has plenty of USDC. To do so, take the following steps:
```
Comment (eshaben) [inline tutorials/eth-api/using-tenderly.md:124]: click Re-Simulate

Comment (eshaben) [reply tutorials/eth-api/using-tenderly.md:124]: oh actually they'll need to click New Simulation

**Review by eshaben — CHANGES_REQUESTED** [review]
such a useful tool! my only concern with this tutorial is how some of it closely resembles the tenderly dev env page

i'll have to think about how we can maybe differentiate the two or possibly update the dev env page to remove some of the examples... not sure right now. lmk if you have any suggestions!

**Comment by themacexpert** [comment]
@eshaben I removed the bullet point section here and linked to the Tenderly page under the builders section to reduce the similarity between the two pages.

**Review by eshaben — APPROVED** [review]
nice job! looks good ✨

**Pull request merged**